docker/Dockerfile

```diff
@@ -161,45 +162,9 @@ ENV NETWORK ${NETWORK:-Mainnet}
 ARG RPC_PORT
 ENV RPC_PORT ${RPC_PORT}
 
-# Set this to log to a file, if not set, logs to standard output
-ARG LOG_FILE
-ENV LOG_FILE ${LOG_FILE}
-
-# Set this to change the default cached state directory
-ARG ZEBRA_CACHED_STATE_DIR
-ENV ZEBRA_CACHED_STATE_DIR ${ZEBRA_CACHED_STATE_DIR:-/var/cache/zebrad-cache}
-
-# Zebra automatically detects if it is attached to a terminal, and uses colored output.
-# Set this to 'true' to force using color even if the output is not a terminal.
-# Set this to 'false' to disable using color even if the output is a terminal.
-ARG LOG_COLOR
-ENV LOG_COLOR ${LOG_COLOR}
```

Comment:
This variable makes sure that deployed Google Cloud instances have readable logs, by disabling colour escape codes. Can you check that still works?

https://github.com/ZcashFoundation/zebra/actions/workflows/continous-delivery.yml

Reply:
Confirmed with deployed instance: https://console.cloud.google.com/compute/instancesDetail/zones/us-east1-c/instances/zebrad-fix-entrypoint-ports-ad5a3a4-mainnet?authuser=1&project=zfnd-dev-zebra